benefits/enrollment/views.py

```diff
@@ -212,17 +212,8 @@ def reenrollment_error(request):
 @decorator_from_middleware(EligibleSessionRequired)
 def retry(request):
     """View handler for a recoverable failure condition."""
-    if request.method == "POST":
-        analytics.returned_retry(request)
-        form = forms.CardTokenizeFailForm(request.POST)
```

Comment:
Does this mean we don't need the `forms.CardTokenizeFailForm` class anymore?

Reply:
Oh wait, maybe we still need it so that the Javascript functions on Eligibility Index can POST back to the server

Reply:
Yeah that is correct, we still use the form for the Javascript POST back -- it just makes that code a little cleaner/easier.

Reply:
This change in the view was just to make this view a little more flexible. There isn't really a good reason to block GET requests. A normal app user should only ever make POST requests (via the Javascript / POST back) but this helps us test the view. And if a user does happen to GET this view... well really no harm there, it does require an eligible session and only sends them back to the index.